bom/pom.xml

```diff
@@ -216,18 +216,18 @@
 
       <dependency>
         <groupId>com.redhat.service.smartevents</groupId>
-        <artifactId>manager-integration-tests</artifactId>
```

Comment:
I think this can be thrown out of the BOM completely

Reply:
Removed.